deployment/ansible/roles/model-my-watershed.geoprocessing/templates/upstart-akka-http.conf.j2

```diff
@@ -0,0 +1,13 @@
+description "Akka HTTP Server"
+
+start on runlevel [3]
+stop on shutdown
+
+kill timeout 20
```

Comment:
Yeah, that bit is just to switch to an unprivileged user, otherwise it'll run as `root`. When I SSH into the old `worker` I had provisioned from before, it has an `mmw` user. This might be a regression on this branch.